package.json

```diff
@@ -24,9 +24,10 @@
     "prepare": "yarn build",
     "start": "yarn storybook",
     "storybook": "start-storybook -p 3000 -s public/development",
-    "test": "jest",
-    "test:watch": "jest --watchAll --runInBand --verbose",
-    "test:update": "jest --updateSnapshot",
+    "test": "jest --no-cache",
+    "test:a11y": "jest --testPathPattern='src\\/([^\\s]+)\\/([a-zA-Z0-9\\s_\\\\.\\-\\(\\):])+(\\.test\\.a11y\\.)+(js|jsx)$'",
+    "test:watch": "jest --watchAll",
+    "test:update": "jest -u",
```

Comment:
These test commands were recently updated in my linting fixes PR #526 and should be

```
"test": "jest",
"test:a11y": "jest --testPathPattern='src\\/([^\\s]+)\\/([a-zA-Z0-9\\s_\\\\.\\-\\(\\):])+(\\.test\\.a11y\\.)+(js|jsx)$'",
"test:watch": "jest --watchAll --runInBand --verbose",
"test:update": "jest --updateSnapshot",
```

Reply:
That is weird. I pulled in from master. will update